terraform/gcp/projects/cluster.tfvars.template

```diff
@@ -22,40 +22,47 @@ regional_cluster = true
 #       Then use that version to explicitly set all k8s versions below, and
 #       finally decomment the k8s_versions section and removing this comment.
 #
-#k8s_versions = {
+# k8s_versions = {
 #  min_master_version : "",
 #  core_nodes_version : "",
 #  notebook_nodes_version : "",
-#  dask_nodes_version : "",
+#  dask_nodes_version : "", # if this cluster will host daskhubs
 #}
 
-core_node_machine_type = "n2-highmem-4"
+core_node_machine_type = "n2-highmem-2"
 
-# Network policy is required to enforce separation between hubs on multi-tenant clusters
-# Tip: uncomment the line below if this cluster will be multi-tenant
-# enable_network_policy  = true
+# Tip: comment the line below if this cluster won't be multi-tenant.
+#       Network policy is required to enforce separation between hubs on
+#       multi-tenant clusters.
+#
+enable_network_policy  = true
 
 # Setup a filestore for in-cluster NFS
 enable_filestore      = true
 filestore_capacity_gb = 1024
 
-user_buckets = {
-  "scratch-{{ hub_name }}" : {
-    "delete_after" : 7
-  },
+# Tip: uncomment and fill the missing info in the lines below if you want
+#       to setup scratch buckets for the hubs on this cluster.
+#
+# user_buckets = {
+#  "scratch-{{ hub_name }}" : {
+#    "delete_after" : 7
+#  },
   # Tip: add more scratch buckets below, if this cluster will be multi-tenant
-}
+#}
 
-hub_cloud_permissions = {
-  "{{ hub_name }}" : {
-    allow_access_to_external_requester_pays_buckets : true,
-    bucket_admin_access : ["scratch-{{ hub_name }}"],
-    hub_namespace : "{{ hub_name }}"
-  },
-  # Tip: add more namespaces below, if this cluster will be multi-tenant
-}
+# Tip: uncomment and fill the missing info in the lines below if you want
+#       to setup specific cloud permissions for the buckets in this cluster.
+#
+# hub_cloud_permissions = {
+#   "{{ hub_name }}" : {
+#     allow_access_to_external_requester_pays_buckets : true,
+#     bucket_admin_access : ["scratch-{{ hub_name }}"],
+#     hub_namespace : "{{ hub_name }}"
+#   },
+#   # Tip: add more namespaces below, if this cluster will be multi-tenant
+# }
```

Comment:
Inconsistent indentation compared to other places (`#` + `space`, instead of just `#`).